system-addon/content-src/components/Topics/_Topics.scss

```diff
@@ -4,36 +4,48 @@
 
   font-size: 13px;
   color: $topic-grey;
-  min-width: 780px;
-  line-height: 16px;
   margin-top: $topic-margin-top;
+  line-height: 1.6;
```

Comment:
like, do you need both the multiplier and also the 16px? i remember the 16px came from an average number on both windows and osx looking pretty good, i wonder if adding the multiplier will change that